### DISCUSSION TITLE
Download Manager v5.1.0.0

### DISCUSSION THREAD

**Discussion by Lilyy2565** [Download-Manager-Community/Download-Manager#93] [Announcements]
This version of Download Manager includes:

- A visual refresh
- Lots of bug fixes
- YouTube Video Downloader and YouTube Playlist Downloader has moved. Just enter the YouTube URL into the Download URL and press Download
- The Current Downloads window now works when downloading file from browser extension
- The Current Downloads window now moves with the main window
- Code signing certificate installation option in installer
- Updated to the latest dependencies

_The release of the portable version of the installer will be delayed again to resolve a Windows Security false positive._

<hr /><em>This discussion was created from the release <a href='https://github.com/Download-Manager-Community/Download-Manager/releases/tag/5.1.0.0'>Download Manager v5.1.0.0</a>.</em>

**Comment by Lilyy2565** [comment]
Update:
The download manager installer portable executable false positive has been removed from Windows Security.
If you have a false positive after this is posted, update to the latest windows security definitions.
The latest release will be updated with the installer portable executable soon.
<img src="https://github.com/Download-Manager-Community/Download-Manager/assets/52353209/527c0bbb-279b-4479-87e1-0a59581ff53f" height="500px" width="auto">

**Comment by wishing1star** [comment]
How to remove Download-Manager?
I installed it in my windows 11. When I choose it in control panel and select remove, it only launches the manager

**Reply by Lilyy2565** [reply]
To uninstall Download Mananger on the latest version of Windows 11 you may use the following methods:
### Settings
- Open Settings > Apps > Installed Apps
- Find Download Manager in the list
- Click on the more options button (3 dots) and then click Uninstall.
<img src="https://github.com/user-attachments/assets/71cd8c1e-2ff3-4e16-81b2-4d8dace54b12" width="500px" height="auto">

- Then click uninstall and Yes to uninstall Download Manager from your system
<img src="https://github.com/user-attachments/assets/015f6736-426d-43cc-87e6-8a17df42aa3e" width="500px" height="auto">

- Wait for the uninstall to complete

### Winget
- Open a command prompt (or powershell) window
- Enter `winget uninstall Soniczac7.DownloadManager`
- Press Yes
- Wait for the uninstall to complete.
<img src="https://github.com/user-attachments/assets/86645085-7a46-43f8-bbe3-47afd39d5ce0" width="700px" height="auto">

### Control Panel
- Open control panel and navigate to Programs and Features
- Find Download Manager in the list of programs and select it
- Click Uninstall/Change
- Click Yes on the open dialog
- Wait for the uninstall to complete
<img src="https://github.com/user-attachments/assets/af3c3e45-cfdb-4d6c-9495-2dac7e9a88b3" width="700px" height="auto">
